blueocean-web/src/main/resources/jenkins/plugins/blueocean/web/Messages_fr.properties

```diff
@@ -3,6 +3,9 @@ login=Connexion
 logout=D\u00e9connexion
 Not.found.heading=Page non trouv\u00e9e (404)
 Not.found.message=Jenkins n'a pas pu trouver la page que vous recherchiez. V\u00e9rifiez l'URL pour des erreurs ou appuyez sur le bouton de retour. 
+parametrised.pipeline.cancel=Annuler
+parametrised.pipeline.header=Input indispensable
```

Comment:
I would say "Entrée(s) indispensable" as "Input" is not a true French word. But the job configuration use the terms "parameters" so we could say "Paramètre(s) indispensable", which would be better for the configuration -> usage understanding.